stratum/hal/lib/common/common.proto

```diff
@@ -1171,6 +1171,16 @@ message FecStatus {
   FecMode mode = 1;
 }
 
+// Wrapper around the SDN port ID.
+// The target is expected to perform accept requests from the controller
+// using the ID assigned by the SDN management system. If the target
+// cannot accept requests using the provided ID, it can provide
+// and alterate ID that should be used by the controller to address
+// the interface.  
+message SdnPortIdOverride {
+  uint32 port_id = 1;
+}
```

Comment:
I wanted to suggest to makes this part of `FrontPanelPortInfo` to make it easy to detect switches that don't set the value, but then I found that `bcm_switch` does not implement that one, too.